### PR TITLE
BDKD-450 Added get-file-list subcommand and python wrapper

### PR DESCRIPTION
**Pull request by lolo-fernandez-sirca**

### CODE REVIEW

File: datastore/lib/bdkd/datastore/util/ds_util.py [hunk 1]
```diff
@@ -81,6 +81,17 @@ def _delete_resource_parser():
                         help='Force deleting a published resource')
     return parser
 
+def _get_file_list_parser(enforce=True):
+    """
+    Parser for get-file-list options.
+    """
+    parser = argparse.ArgumentParser(add_help=False)
+    optional_fields = parser.add_argument_group('Optional arguments',
```
Comment (balram-ramanathan-sirca) [inline datastore/lib/bdkd/datastore/util/ds_util.py]: Any reason to use add_argument_group? Just calling add_argument to parser should be enough for a single argument.

**Comment by balram-ramanathan-sirca** [comment]
One minor suggestion, otherwise looks good.

**Comment by lolo-fernandez-sirca** [comment]
@sirca/back-up Suggestion implemented.

**Comment by balram-ramanathan-sirca** [comment]
Looks good, merging

**Pull request merged**